README.md

```diff
@@ -1,27 +1,32 @@
 Geoserver Build
 ===============
 
-Configures a GeoServer war file with the following;
+Build a GeoServer war file with the following;
 
 * The XSLT extension installed
 * CSV with metadata header plugin
 * Layer filter extension
+* Netcdf output extension
 
-To build:
-
+### To build:
 ```
 git submodule update --init
-mvn clean install -U -Dmaven.test.skip=true 
+mvn clean
+mvn -P wps,xslt install -DskipTests
```

Comment:
No real reason to use install here - package will build all the artifacts.  We don't use local repositories for artifacts we build.

I'd remove -Dskiptests - we don't want to encourage building without testing.  Thats something the developer can choose to do if they want to speed up building while developing if they know what they are doing.